.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           go-version-file: go.mod
 
       - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v5
+        uses: goreleaser/goreleaser-action@v6
         with:
           version: latest
```

Comment:
Could you update the version constraint here and in the `release.yml` as mentioned in the upgrade docs? https://goreleaser.com/blog/goreleaser-v2/?h=v2#github-action

Reply:
Sorry. What do you mean?

> If you use our [GitHub Action](https://github.com/goreleaser/goreleaser-action), the latest version (v6.0.0) should use ~> v2 by default if your version option is either empty or latest.

If `version` is `latest`, the version constraint `~> v2` is used.

Reply:
Do you prefer `version: "~> v2"` rather than `version: "latest"`?

Reply:
Yes, we want to pin the version to v2. The blog post describes how there will be more major releases (and therefor more breaking changes) in the future, so we want to make sure we don't accidentally break our release pipeline.

I personally prefer the `version: 2.x` format, but that's up to you.

Reply:
> I personally prefer the version: 2.x format, but that's up to you.

I prefer pinning the full version.

```yaml
version: v2.0.1
```

Reply:
And ideally we should update the version by Renovate.
We can do this by custom manager.
https://docs.renovatebot.com/modules/manager/regex/

Reply:
I'd prefer to not pin the full version, we should be consistent with our other projects.

Reply:
I see. Fixed. 08f773d353df4fa4f3d0d74a98b99cfef24a324e